tsl/src/bgw_policy/job.h

```diff
@@ -15,6 +15,9 @@
 #include "continuous_aggs/materialize.h"
 #include "bgw_policy/chunk_stats.h"
 
+/* Add config keys common across job types here */
+#define CONFIG_KEY_VERBOSE_LOG "verbose_log" /*used only by compression now*/
```

Comment:
can we call this verbose?

Reply:
I use verbose_log so that it is clear to the user that output is written to the log files.